yt_dlp/extractor/pokergo.py

```diff
@@ -0,0 +1,110 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import base64
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    try_get,
+)
+
+
+class PokerGoBaseIE(InfoExtractor):
+    _NETRC_MACHINE = 'pokergo'
+    _AUTH_TOKEN = None
+    _PROPERTY_ID = '1dfb3940-7d53-4980-b0b0-f28b369a000d'
+
+    def _login(self):
+        username, password = self._get_login_info()
+        if not username:
+            self.raise_login_required(method='password')
+
+        self.report_login()
+        self._AUTH_TOKEN = self._download_json(
```

Comment:
```suggestion
        PokerGoBaseIE._AUTH_TOKEN = self._download_json(
```
This will make sure login happens only once for both extractors combined